MetaMask/metamask-mobile
# fix: metrics event not fired when blockaid is disabled

## **Description**
Metrics event not fired when blockaid is toggled off

## **Related issues**

Fixes: MetaMask/metamask-mobile#8064

## **Manual testing steps**

1. Go to settings page
2. Enable and then disable blockaid
3. Check that metrics event if fired

## **Screenshots/Recordings**
NA

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
